python/lsst/ts/dream/common/abstract_dream.py

```diff
@@ -0,0 +1,78 @@
+# This file is part of ts_dream.
+#
+# Developed for the Vera C. Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["AbstractDream"]
+
+from abc import ABC, abstractmethod
+import typing
```

Comment:
really picky comment, but consider import typing before abc... 

```
import typing

from abc import ABC, abstractmethod
```

maybe it is just me but I usually prefer to have import organized such that simple imports comes first than "from" and so on...

Reply:
Done